exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/writer/TestParquetWriter.java

```diff
@@ -969,12 +969,15 @@ public void testInt96TimeStampValueWidth() throws Exception {
     try {
       testBuilder()
           .ordered()
-          .sqlQuery("select c, d from cp.`parquet/data.snappy.parquet` where d = '2015-07-18 13:52:51'")
+          .sqlQuery("select c, d from cp.`parquet/data.snappy.parquet` " +
+              "where `a` is not null and `c` is not null and `d` is not null")
```

Comment:
If you have several baseline values and indicate that output is ordered `order by` clause must be present in query to ensure that result will always be returned in expected order, otherwise use `unOrdered()`.